libenkf/src/enkf_main_jobs.c

```diff
@@ -538,21 +538,45 @@ void * enkf_main_analysis_update_JOB( void * self , const stringlist_type * args
   return NULL;
 }
 
-
+// SHould skip compltely if data_root is unset.
 void * enkf_main_pre_simulation_copy_JOB( void * self , const stringlist_type * args) {
   const char * source_path  = stringlist_iget( args , 0 );
 
-  if (util_is_directory( source_path )) {
-    enkf_main_type * enkf_main = enkf_main_safe_cast( self );
-    model_config_type * model_config = enkf_main_get_model_config( enkf_main );
-    if (stringlist_get_size( args ) == 2)
-      model_config_set_data_root( model_config , stringlist_iget(args, 1) );
+  if (!util_entry_exists( source_path )) {
+    char * msg = util_alloc_sprintf("Error in workflow job PRE_SIMULATION_COPY - source argument: %s not existing\n",source_path);
+    res_log_add_message(LOG_ERROR, stderr , msg , true );
```

Comment:
Please `free(msg)` in both of the logging calls

Reply:
Actually the `res_log_add_message( )` discards the message - that is the last `true`  in the argument list. Yes - it is a funny api.